src/scales/scale.radialLinear.js

```diff
@@ -208,9 +208,28 @@ function drawPointLabels(scale, labelCount) {
     const {backdropColor} = optsAtIndex;
 
     if (!isNullOrUndef(backdropColor)) {
+      const borderRadius = toTRBLCorners(optsAtIndex.borderRadius);
```

Comment:
I had a default value of `0` added for this new setting after line 606, but then the option failed to work. The proxy always showed the default value instead of what was specified. Not sure why that was the case, but leaving no default also worked